commons/aws_s3.py

```diff
@@ -0,0 +1,33 @@
+import boto3
+from botocore.exceptions import ClientError
+from storages.backends.s3boto3 import S3Boto3Storage
+from django.conf import settings
+
+
+class awsS3:
+    def create_aws_client(self):
+        return boto3.resource("s3", endpoint_url=settings.AWS_S3_HOST)
+
+    # For binary files
+    def upload_file_obj(self, file, mykey):
+        s3 = self.create_aws_client()
```

Comment:
Still need to update this class in order to reduce repeated lines of code (I think is with def  __init__(self) )